core/aclmgmt/resources/resources.go

```diff
@@ -51,6 +51,7 @@ const (
 	Cscc_JoinChainBySnapshot  = "cscc/JoinChainBySnapshot"
 	Cscc_JoinBySnapshotStatus = "cscc/JoinBySnapshotStatus"
 	Cscc_GetConfigBlock       = "cscc/GetConfigBlock"
+	Cscc_GetChannelConfig     = "cscc/GetChannelConfig"
```

Comment:
I grepped "GetConfigBlock" to ensure we make all the associated updates.
It looks like you got the code references, but these other test, doc, and sample files should also be updated when introducing a new acl resource:
integration/e2e/acl_test.go
docs/source/access_control.md
sampleconfig/configtx.yaml

Reply:
1) Added a call in acl_test.go (BTW, I noticed only GetConfigBlock from cscc. I guess that this file needs a revisit)
2) This grep was just a reference as an example. Do you know if we are documenting these APIs somewhere or only end CLI is documented?
3) I am not sure if we are consistently updating this (I see many missing here as well), but added an entry for this new function

Reply:
2. I believe only the end CLI is documented.